docs/checkit.js

```diff
@@ -817,10 +770,26 @@ var Calendar = function(state, checkitObj) {
     //user wants to track
     self.numberOfMonths = self.endDate.diff(self.startDate, 'months', true);
     self.monthObjects = self.generateMonthObjects();
-    self.checkit = checkitObj;
-    
+    // Last month to be added, this is so that when you need to attach the
+    // click handler to this month, you don't do it to the entire calendar.
+    self.lastMonth = null;
```

Comment:
You probably don't need this. At the very least you can always just grab it manually from the `monthObjects` array.